### PR TITLE
fix: solve prop to state warning from react >= 16.5.0

### PR DESCRIPTION
**Pull request by eljefedelrodeodeljefe**
On a fresh install of all dependencies a setup like this will print annoying error messages from react DOM.

A fix should be making a shallow copy, as taken from the react issue tracker. I am not so confident about the consequences for this library and would rely on your guidance.

```html
<template>
  <div>
    <three :passed-props="passedProps" />
  </div>
</template>

<script>
import React from 'react'

class Three extends React.Component {
  render () {
    return (
      <div>
        React
      </div>
    )
  }
}

export default {
  name: 'HelloWorld',
  components: { three: Three },
  props: {
    msg: {
      type: String,
      required: true
    }
  },
  computed: {
    passedProps () {
      return {
        message: this.message,
        reset: this.reset
      }
    }
  }
}
</script>
```

### CODE REVIEW

**Comment by eljefedelrodeodeljefe** [comment]
breaks liniting. Will fix if this is wanted.

**Comment by akxcv** [comment]
Hi @eljefedelrodeodeljefe and thank you for this pull request! This is looking good and I'll merge this if you fix the lint error.

And sorry for taking such a long time!

File: src/wrappers/React.js [hunk 1]
```diff
@@ -10,11 +10,13 @@ const makeReactContainer = Component => {
       super(props)
 
       /**
-       * We create a stateful component in order to attach a ref on it. We will use that ref to
-       * update component's state, which seems better than re-rendering the whole thing with
+       * We previously created a stateful component in order to attach a ref on it. We have used that ref to
```
Comment (akxcv) [inline src/wrappers/React.js]: Actually, lines 13 and 14 shouldn't be changed. Comments there are not related to the issue.

**Comment by akxcv** [comment]
Fixed in #62

**Pull request closed without merging**